nidirect_common/tests/src/Nightwatch/Tests/contentTypeValidation.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+var types = [];
+const configDir = '../../config/sync';
+
+module.exports = {
+  '@tags': [
+    'nidirect',
+    'nidirect_common',
+  ],
+
+  'Test whether content types found in config folder are present in the UI': browser => {
+    browser
+      .pause(9000, function () {
+        fs.readdirSync(configDir).forEach(file => {
+          if (match = file.match(/node.type.(\w+)/)) {
+            // Open a readstream to the file.
+            var content = fs.readFileSync(configDir + '/' + file, 'utf8');
+            // Convert the string with line breaks into an array to ease
+            // iteration/regex matching without having to work around single quotes
+            // or control characters nestled alongside the values we want to try and extract.
+            lines = content.split(/\n/);
+
+            for (var i = 0; i < lines.length; i++) {
+              // Drop single quotes as they make word matching/detection much harder.
+              lines[i] = lines[i].replace(/'/g,'');
+              matches = lines[i].match(/^name: (.+)/);
+
+              if (matches) {
+                types.push(matches[1]);
+              }
+            }
+          }
+        });
+
+        browser
+          .drupalLogin({name: process.env.TEST_USER, password: process.env.TEST_PASS})
+          .drupalRelativeURL('/admin/structure/types')
+          .waitForElementVisible('body', 1000)
```

Comment:
I'd be tempted to raise this pause a little, I've had other tests fail due to body load speeds.

Reply:
Yep, makes sense. It now matches the pause times in the migration modules.